plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -85,6 +87,13 @@ private boolean validateLocalFields() {
       return false;
     }
     
+    File parent = getLocationPath().toFile();
+    File projectDirectory = new File(parent, getProjectName());
```

Comment:
Actually, checking this now it looks like we're ignoring the custom location completely and always putting it into the workspace default.